### PR TITLE
*DO NOT MERGE* MS-734 Do they want their case referred to NRM? Page 

### PR DESCRIPTION
**Pull request by jjrobothamho**
Do they want their case referred to NRM? Page + acceptance tests

### CODE REVIEW

File: apps/nrm/views/pv-want-to-submit-nrm.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{{<partials-page}}
+  {{$page-content}}
+    <p>If a potential victim wants you to refer their case, they can get:</p>
+    <ul class="govuk-list govuk-list--bullet">
+        <li>access to support (for example, housing and legal advice)</li>
+        <li>a formal decision – this can help if they’re being prosecuted for a crime they were forced to commit</li>
+    </ul>
+    <div class="govuk-inset-text">
+        We will need to share their personal details with organisations such as the police and the Home Office if they refer their case.
```
Comment (sulthan-ahmed) [inline apps/nrm/views/pv-want-to-submit-nrm.html]: Can you put the content in the translations?

File: apps/nrm/views/partials/pv-want-to-submit-nrm-no.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<fieldset id="pv-want-to-submit-nrm-no-fieldset">
+    <div class="govuk-radios__conditional">
+            <p>They understand this means their case will not be referred to the NRM and they will not receive support or a formal decision.</p>
```
Comment (sulthan-ahmed) [inline apps/nrm/views/partials/pv-want-to-submit-nrm-no.html]: Can you put the content in the translations?

File: acceptance-test/user-pathways/upload-file/upload-file.test.js [hunk 3]
```diff
@@ -100,84 +265,56 @@ describe('Upload File(s)', () => {
...
             // reported-to-police
-            await page.waitForSelector(PV_HAS_CRIME_REFERENCE_NUMBER_YES_OPTION);
-            await page.click(PV_HAS_CRIME_REFERENCE_NUMBER_YES_OPTION);
-            await clickContinueButton(1);
+            await doTheyHaveACrimeReferenceNumberPage();
 
-            // Run through the skeleton until we reach the upload page
-            await clickContinueButton(13);
+            // pv-want-to-submit-nrm
+            await doTheyWantTheirCaseReferredToTheNrmPage();
```
Comment (sulthan-ahmed) [inline acceptance-test/user-pathways/upload-file/upload-file.test.js]: Can you put some of these all in one function and just pass in selector and value

File: acceptance-test/user-pathways/happy-path/happy-path.test.js [hunk 4]
```diff
@@ -53,103 +64,106 @@ describe('Critical user path(s)', () => {
...
-     * @returns {void}
+     * @returns {Promise}
      */
-    async function whoDoYouWorkForPage() {
-        await page.waitForSelector(ORGANISATION_INPUT);
+    async function verifyUser() {
+        // start
+        await clickElement(CONTINUE_BUTTON);
+        // who-do-you-work-for
         await page.$eval(ORGANISATION_INPUT, (element) => {
```
Comment (jjrobothamho) [inline acceptance-test/user-pathways/happy-path/happy-path.test.js]: The element.value cannot be overwritten by a nodeJS variable as the context in which this section of code will be run is in the browser and has no access to the node variables (the $eval function)

hence the string will need to be stated explicitly
@sulthan-ahmed 

other functions  have been converted to functions

**Comment by sulthan-ahmed** [comment]
LGTM just need to skip that test

**Pull request closed without merging**